valinta-tulos-valintarekisteri-db/src/main/scala/fi/vm/sade/valintatulosservice/tarjonta/HakuService.scala

```diff
@@ -203,7 +204,7 @@ class TarjontaHakuService(config: HakuServiceConfig) extends HakuService with Js
     }.right.flatMap(_.toRight(new IllegalArgumentException(s"No hakukohde found for haku $hakuOid")))
   }
   def getHakukohdes(oids: Seq[String]): Either[Throwable, List[Hakukohde]] = {
-    sequence(for{oid <- oids.toStream} yield getHakukohde(oid))
+    MonadHelper.sequence(for{oid <- oids.toStream} yield getHakukohde(oid))
```

Comment:
diggaisin jos noiden lokkien ympärillä olis välilyönnit kautta linjan :) Eli 

`MonadHelper.sequence(for { oid <- oids.toStream } yield getHakukohde(oid))`